recipes/pyquickhelper/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyquickhelper" %}
+{% set version = "1.10.3585" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyquickhelper-{{ version }}.tar.gz
+  sha256: 98e999d24f5cba90d72a996e80dfd37294b8ef6ecc40487855071c1293674f0b
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - encrypt = pyquickhelper.cli.encryption_cli:encrypt
+    - decrypt = pyquickhelper.cli.encryption_cli:decrypt
+    - encrypt_file = pyquickhelper.cli.encryption_file_cli:encrypt_file
+    - decrypt_file = pyquickhelper.cli.encryption_file_cli:decrypt_file
+    - pyq-sync = pyquickhelper.cli.pyq_sync_cli:pyq_sync
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pyquicksetup
+    - python >=3.6
+  run:
+    - fire
+    - numpy >=1.16.0
+    - pyquicksetup >=0.2
+    - python >=3.6
+
+test:
+  imports:
+    - pyquickhelper
+    - pyquickhelper.benchhelper
+  commands:
+    - pip check
+    - encrypt --help
+    - decrypt --help
+    - encrypt_file --help
+    - decrypt_file --help
+    # - pyq-sync --help  # currently broken: https://github.com/sdpython/pyquickhelper/issues/340
+  requires:
+    - pip
+
+about:
+  home: http://www.xavierdupre.fr/app/pyquickhelper/helpsphinx/index.html
+  summary: 'Various functionalities: folder synchronization, simple logging function, helpers to generate documentation with sphinx, sphinx extension, to run a command line, to run a notebook...'
```

Comment:
Can you make this a little shorter? The `summary` should be limited to ~80 characters while longer text can be included in the `description` field.

Reply:
I have changed the summary, according to what it says in the title of the [pyquickhelper homepage](http://www.xavierdupre.fr/app/pyquickhelper/helpsphinx/).

Reply:
I have changed the summary according to what it says in the title of the [pyquickhelper homepage](http://www.xavierdupre.fr/app/pyquickhelper/helpsphinx/).